### PR TITLE
MINOR: doc: note how JDK-8136913 can affect client SASL

### PR DESCRIPTION
**Pull request by tombentley**

### CODE REVIEW

**Comment by fvaleri** [comment]
LGTM. This can spare some headaches.

File: docs/security.html [hunk 1]
```diff
@@ -677,7 +677,11 @@ <h3 class="anchor-heading"><a id="security_sasl" class="anchor-link"></a><a href
                     <p>To configure SASL authentication on the clients, select a SASL
                         <a href="#security_sasl_mechanism">mechanism</a> that is enabled in
                         the broker for client authentication and follow the steps to configure SASL
-                        for the selected mechanism.</p></li>
+                        for the selected mechanism.</p>
+
+                    <p>Note: Due to how the JRE implements reverse DNS lookups, clients may observe 
+                        slow SASL handshakes if fully qualified domain names are not used when configuring a broker's
```
Comment (mimaison) [inline docs/security.html]: What about adding `When establishing connections to brokers via SASL, clients perform a reverse DNS lookup of the provided broker addresses.` just before?

Comment (tombentley) [reply docs/security.html]: Thanks, I reworded it slightly. Let me know what you think.

**Review by mimaison — APPROVED** [review]
LGTM

**Pull request merged**